kura/org.eclipse.kura.core.configuration/src/main/java/org/eclipse/kura/core/configuration/ConfigurationServiceImpl.java

```diff
@@ -1456,4 +1545,19 @@ private Tocd getOCDForPid(String pid) {
         }
         return ocd;
     }
+
+    /**
+     * Convert property value to string
+     * @param value the input value
+     * @return the string property value, or {@code null}
+     */
+    private static String makeString(Object value) {
+		if ( value == null ) {
+			return null;
+		}
+		if ( value instanceof String ) {
+			return (String)value;
```

Comment:
It would only work if and only if the value is of type `String`. I would suggest you to use `String.valueOf()` instead.